packages/gatsby-theme-bodiless/src/dist/GatsbyMobxStore.ts

```diff
@@ -290,4 +150,25 @@ export default class GatsbyMobxStore {
       this.setItem(key, new Item(this, key, value, event));
     }
   };
+
+  getChildrenNodes = (keyPath: string[]) => {
+    const key = keyPath.join(nodeChildDelimiter);
+    const children = Array.from(this.store)
+      .filter(item => item[0].indexOf(key) === 0 && item[0] !== key);
```

Comment:
Why do we bother excluding the key here when we just have to make a separate call to delete the parent later?